### PR TITLE
Add Check labels feature

### PR DESCRIPTION
**Pull request by Shmuel-columbia**

### CODE REVIEW

File: lightning_pose_app/ui/extract_frames.py [hunk 1]
```diff
@@ -825,3 +903,221 @@ def _render_streamlit_fn(state: AppState):
 
             #     #force rerun to show "waiting for existing..." message
             st_autorefresh(interval=2000, key="refresh_extract_frames_after_submit")
+
+    # Custom CSS for styling
+    st.markdown(
```
Comment (themattinthehatt) [inline lightning_pose_app/ui/extract_frames.py:908]: are all of these entries necessary? ok if they are, but this is a lot of code and we should only include the minimum required

File: lightning_pose_app/ui/extract_frames.py [hunk 2]
```diff
@@ -825,3 +903,221 @@ def _render_streamlit_fn(state: AppState):
...
+        st.caption("This button will be enabled once labeled frames are available for review.")
+        if st_start_check_labels:
+            state.st_check_status = "initialized"
+            st.text("Request submitted!")
+            state.selected_body_parts = selected_body_parts
+            state.run_script_check_labels = True
+
+        st_autorefresh(interval=2000, key="refresh_check_labels")
+
+        labeled_data_check_path = os.path.join(state.proj_dir[1:], 'labeled-data-check')
```
Comment (themattinthehatt) [inline lightning_pose_app/ui/extract_frames.py:1049]: ah yes, good reason to change "labeled-data-check" to a variable that is imported

File: lightning_pose_app/ui/streamlit_video_viewer.py
Comment (themattinthehatt) [inline lightning_pose_app/ui/streamlit_video_viewer.py:1]: can remove `get_models` defined on line 30 and import the `find_models` function from the project backend

File: lightning_pose_app/ui/streamlit_video_viewer.py [hunk 3]
```diff
@@ -78,77 +78,85 @@ def _render_streamlit_fn(state: AppState):
...
-        results_files = list_results_files(video_name, model_dir, selected_model)
-
-        if results_files:
-            selected_result_files = st.multiselect(
-                "**Step 3:** Select results files to download", list(results_files.keys())
+    if os.path.exists(model_dir):
+        st.header("Visualize Model Predictions")
+        with st.sidebar:
+            st.write("Select the model and then the associated video you wish to inspect")
+            trained_models = get_models(model_dir)  # create a list of all models
```
Comment (themattinthehatt) [inline lightning_pose_app/ui/streamlit_video_viewer.py:85]: see file-level comment about replacing this with the `find_models` function from the project backend

**Review by themattinthehatt — CHANGES_REQUESTED** [review]

File: lightning_pose_app/ui/extract_frames.py (hunk 1 above)
Comment (Shmuel-columbia) [reply lightning_pose_app/ui/extract_frames.py:908]: yes. streamlit needs long text to set all those viz parmeters

File: lightning_pose_app/ui/extract_frames.py [hunk 4]
```diff
@@ -629,7 +707,7 @@ def _render_streamlit_fn(state: AppState):
                     p = 100.0
                 else:
                     st.text(status)
-                st.progress(p / 100.0, f"{vid} progress ({status}: {int(p)}\% complete)")
+                st.progress(p / 100.0, f"{vid} progress ({status}: {int(p)}% complete)")
```
[the remark this answers is not in the document]
Comment (Shmuel-columbia) [reply lightning_pose_app/ui/extract_frames.py:710]: works fine for me

**Review by themattinthehatt — APPROVED** [review]

**Pull request merged**